### DISCUSSION TITLE
Can I mine this coin with My LG Electronics TRIPLE XD TV?

### DISCUSSION THREAD

**Discussion by lchrk241721** [revoxhere/duino-coin#1430] [Ideas]
Hello guys. I have a small question here. Can I mine this coin with My LG Electronics TRIPLE XD TV? If answer is yes, then how to do it? Can anyone guide me please?

**Comment by LDarki** [comment]
Hi, the most easy way is using the webminer.
Run the browser of the smart tv and put this url: [https://server.duinocoin.com/webminer.html](https://server.duinocoin.com/webminer.html)
